jsonschema-core.md

```diff
@@ -523,11 +523,29 @@ to inform the user of the evaluation path that produced the error or annotation.
 
 ### Keyword Interactions
 
-Keyword behavior MAY be defined in terms of the annotation results of
-[subschemas](#root) and/or adjacent keywords (keywords within the same schema
-object) and their subschemas. Such keywords MUST NOT result in a circular
-dependency. Keywords MAY modify their behavior based on the presence or absence
-of another keyword in the same [schema object](#schema-document).
+Unless otherwise specified, keywords act independently.
+
+Keywords MAY modify their behavior based on the presence, absence, or value of
+another keyword in the same schema object. Such keywords MUST NOT result in a
+circular dependency.
```

Comment:
It seems unexpected to have a MUST NOT here about this. It's not wrong, but pretty much all the other MUSTs are either what implementations must do, or what schemas must conform to. This is something another vocabulary or keyword _specification_ must do, so only relevant for a minuscule fraction of readers. Not that it shouldn't be here, but maybe some clarification of the context, what this MUST applies to, would help.

Reply:
This was already present, I just flipped the sentences because it made more sense to me and felt less redundant this way.

Reply:
> Not that it shouldn't be here, but maybe some clarification of the context, what this MUST applies to, would help.

I have a bigger issue with the flow of the document, and I intend to make some bigger changes to the organization.  I think this is better saved for when I tackle that.

Reply:
Sounds good, no objection to that.